core/src/main/java/tech/pegasys/web3signer/core/Runner.java

```diff
@@ -186,48 +183,62 @@ public static OpenAPI3RouterFactory getOpenAPI3RouterFactory(
           }
         });
 
-    final OpenAPI3RouterFactory openAPI3RouterFactory = completableFuture.get();
+    final RouterBuilder routerBuilder = completableFuture.get();
 
     // disable automatic response content handler as it doesn't handle some corner cases.
     // Our handlers must set content type header manually.
-    openAPI3RouterFactory.getOptions().setMountResponseContentTypeHandler(false);
-    return openAPI3RouterFactory;
+    routerBuilder.getOptions().setMountResponseContentTypeHandler(false);
+    routerBuilder.getSchemaParser().withStringFormatValidator("uint64", Runner::validateUInt64);
```

Comment:
lets add a comment as why we are adding a uint64  format validator.

Reply:
done